rnd/autogpt_server/autogpt_server/blocks/wikipedia.py

```diff
@@ -0,0 +1,33 @@
+import requests
+from autogpt_server.data.block import Block, BlockSchema, BlockOutput
+
+class GetWikipediaSummary(Block):
+    class Input(BlockSchema):
+        topic: str
+
+    class Output(BlockSchema):
+        summary: str
+
+    def __init__(self):
+        super().__init__(
+            id="h5e7f8g9-1b2c-3d4e-5f6g-7h8i9j0k1l2m",
+            input_schema=GetWikipediaSummary.Input,
+            output_schema=GetWikipediaSummary.Output,
+            test_input={"topic": "Artificial Intelligence"},
+            test_output={"summary": "Artificial intelligence (AI) is intelligence demonstrated by machines, in contrast to the natural intelligence displayed by humans and animals."},
```

Comment:
This is a pair, also the text can in the wiki pedia can change, you can simply assert the type, e.g:

test_output=("summary", str)